core/arch/arm/kernel/secure_partition.c

```diff
@@ -594,6 +681,12 @@ static TEE_Result sp_init_uuid(const TEE_UUID *uuid, const void * const fdt)
 	if (res)
 		goto out;
 
+#ifdef CFG_CORE_TPM_EVENT_LOG
+	res = handle_tpm_event_log(fdt_copy, sess);
```

Comment:
nit: instead of `sess` you could pass `ctx` like in `handle_fdt_dev_regions()`

Reply:
Done